index.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable import/no-cycle */
+import displayItem from './modules/displayItem.js';
+import displayBooks from './modules/displayBooks.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+export const form = document.querySelector('form');
+export const bookTitle = document.querySelector('.title');
+export const bookAuthor = document.querySelector('.author');
+
+displayItem();
+
+const dateTime = DateTime.local();
+const formattedDateTime = dateTime.toLocaleString(DateTime.DATETIME_FULL);
+document.getElementById('date').innerHTML = formattedDateTime;
+
+(function onReload() {
+  if (localStorage.getItem('books')) {
+    const allBooks = JSON.parse(localStorage.getItem('books'));
+    displayBooks(allBooks);
+  }
```

Comment:
- [ ] Please refactor this functions to arrow functions as requested in the project requirement.
![image](https://github.com/marvin-nyalik/Awesome-Books-ES6/assets/91142432/dde3f763-39a6-484d-a923-d03ed0820348)